### PR TITLE
Urf frontliner fixes

### PR DESCRIPTION
**Pull request by lngredients**
Minor fixes of typos and stats

### CODE REVIEW

**Comment by lngredients** [comment]
I'm getting a mob_movement error, but I think that was happening before and is known, Otherwise if it passes everything else it should be good to go.

**Comment by lngredients** [comment]
Please review if possible.

**Comment by lngredients** [comment]
They were getting to be numerous and so I deleted the old ones, I thought they automatically merged into one big changelog file and so could be deleted?

**Comment by bloxgate** [comment]
The merging isn't automatic, its done periodically by someone with the ability to do a direct push.

Hopefully it'll be moved over to Cicero or a similar bot in the near future though.

**Review by caelaislinn — APPROVED** [review]
Item descriptions are a bit wordy but that's ok

**Comment by lngredients** [comment]
@caelaislinn I like my items to have BACKSTORY BRO!

File: html/changelogs/Ingredients New Floor signs 12 July.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - imageadd: "New floorsigns for medical."
```
Comment (caelaislinn) [inline html/changelogs/Ingredients New Floor signs 12 July.yml]: These changes are in your other PR not here, so this changelog entry needs to be removed from this PR. If you're working in a separate branch for each PR you should avoid this issue

**Review by caelaislinn — DISMISSED** [review]

File: html/changelogs/Ingredients New Floor signs 12 July.yml (hunk 1 above)
Comment (lngredients) [reply html/changelogs/Ingredients New Floor signs 12 July.yml]: I was just adding in all the changelogs i could find, I'll take that one out though.

**Review by BDpuffy420 — APPROVED** [review]
